rpcs3/rpcs3qt/progress_dialog.cpp

```diff
@@ -2,10 +2,12 @@
 
 #include <QLabel>
 
-progress_dialog::progress_dialog(const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
+progress_dialog::progress_dialog(const QString &windowTitle, const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
 	: QProgressDialog(labelText, cancelButtonText, minimum, maximum, parent, flags)
 {
+	setWindowTitle(windowTitle);
 	setFixedWidth(QLabel("This is the very length of the progressdialog due to hidpi reasons.").sizeHint().width());
+	setValue(0);
```

Comment:
I think setting the value to 0 actually moves the dialog from an idle state to a running state which might not be intended. (I don't recall the exact process)